doc/howto/move_instances.md

```diff
@@ -54,11 +54,20 @@ For containers, there is limited support.
 Virtual machines can be moved to another server while they are running, thus without any downtime.
 
 To allow for live migration, you must enable support for stateful migration.
-To do so, ensure the following configuration:
+To do so, ensure the following configuration if your instance is not on a shared storage backend:
```

Comment:
Why the mention of shared storage here? This isn't related to size.state, the instances.migration.stateful setting is storage pool agnostic.